pom.xml

```diff
@@ -358,7 +359,20 @@
                 <scala.version>${scala_2.13.version}</scala.version>
             </properties>
         </profile>
-
+        <profile>
+            <id>spark-3.0</id>
+            <properties>
+                <spark.version>3.0.0</spark.version>
```

Comment:
There is also spark v 3.0.1. 
 Release notes - https://spark.apache.org/releases/spark-release-3-0-1.html